components/Chips/examples/ChipsInputExampleViewController.m

```diff
@@ -29,31 +29,33 @@ @implementation ChipsInputExampleViewController {
 - (id)init {
   self = [super init];
   if (self) {
-    _colorScheme =
-        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-    _typographyScheme = [[MDCTypographyScheme alloc] init];
-    _shapeScheme = [[MDCShapeScheme alloc] init];
+    self.containerScheme = [[MDCContainerScheme alloc] init];
+    ;
```

Comment:
Here and throughout!

Reply:
Yes, what happened is I copy/pasted a line with an extra semicolon then clang format moved it to a new line everywhere haha